src/test/java/Handler/RootHandlerTest.java

```diff
@@ -0,0 +1,32 @@
+import static org.junit.Assert.assertEquals;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RootHandlerTest {
+  private Request goodRequest;
+  private String jarDirectoryPath;
+  
+    @Before 
+    public void setup() {
+      
+      String rootPath = System.getProperty("user.dir");
+      this.jarDirectoryPath = rootPath + "/build/libs";
+      
+      this.goodRequest = new Request.Builder()
+                                    .method("GET")
+                                    .uri("/")
+                                    .version("1.1")
+                                    .build();
+    }
+  
+    @Test
+    public void returnContentsOfDirectory() {
```

Comment:
This test is failing when I run the tests on my machine. Unit tests should be portable and repeatable - a test should not depend on the environment in which it is running. 

As part of the setup for this test, we can create a temporary directory with some temporary dummy files to use as the path that we will be injecting into our handler. By doing this, we can be certain that we will see the same output no matter where this test is being run. Be sure to delete any temporary data as part of the teardown of the tests.

Reply:
I've configured the setup of the test to create a directory, populate it with two files, run the test, and delete the files and directory. The setup method is currently 25 lines. I'm not sure how to reduce it significantly without the use of void methods to create the directory and each file.